### ISSUE TITLE
use CLOCK_BOOTTIME on Linux where applicable

### ISSUE THREAD

**Issue by Hello71** [mobile-shell/mosh#889]
I think at least the "last seen" timer should be changed, so that it more accurately reflects stopwatch time.

**Comment by cgull** [comment]
The thing I'm thinking of (but have not implemented) is that for longer delays, mosh-client should report the time **at** which the remote was last heard, rather than the time **since**-- something like "Last contact at 2017/05/15 13:59."

**Comment by Hello71** [comment]
DST?

**Comment by cgull** [comment]
True, time zone changes are a problem with that idea.  And what time format should mosh-client use?  An international standard, or a localized one?

**Comment by Hello71** [comment]
the rest of the string is localized, so it would seem to make sense to localize the time too.
